gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
```

Comment:
perhaps there should be spacings between the variables and the value?

like distributionBase = GRADLE_USER_HOME and the sorts?